projects/storefrontlib/src/cms-components/configurator/generic/configure-cart-entry/configure-cart-entry.component.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { I18nTestingModule } from '@spartacus/core';
+import { ConfigureCartEntryComponent } from './configure-cart-entry.component';
+
+@Pipe({
+  name: 'cxUrl',
+})
+class MockUrlPipe implements PipeTransform {
+  transform(): any {}
+}
+
+describe('ConfigureCartEntryComponent', () => {
+  let component: ConfigureCartEntryComponent;
+  let fixture: ComponentFixture<ConfigureCartEntryComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [I18nTestingModule, RouterTestingModule],
+      declarations: [ConfigureCartEntryComponent, MockUrlPipe],
+      providers: [],
+    }).compileComponents();
+  }));
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ConfigureCartEntryComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should create component', () => {
+    expect(component).toBeDefined();
+  });
+});
```

Comment:
Not sure, but: Could we maybe also test the navigation? (I didn't look into this, also for configure-product, but it looks it's possible triggering a navigation and then check the location object)